x-pack/plugins/alerting/server/routes/rule/apis/bulk_enable/transforms/transform_bulk_enable_response/v1.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { BulkEnableRulesResponseV1 } from '../../../../../../../common/routes/rule/apis/bulk_enable';
+import { RuleParamsV1 } from '../../../../../../../common/routes/rule/response';
+import { BulkEnableRulesResult } from '../../../../../../application/rule/methods/bulk_enable/types';
+import { transformRuleToRuleResponseV1 } from '../../../../transforms';
+import { Rule, RuleParams } from '../../../../../../application/rule/types';
+
+export const transformBulkEnableResponse = <Params extends RuleParams = never>(
+  response: BulkEnableRulesResult<Params>
+): BulkEnableRulesResponseV1<RuleParamsV1>['body'] => {
+  return {
+    rules: response.rules.map((rule) => transformRuleToRuleResponseV1<Params>(rule as Rule)),
```

Comment:
I noticed that before the attributes we were returning were camel case but now it is snake case. Is this considered a breaking change?

Reply:
Nah we were always returning snake case, the transform was done in at the route level thats all.

Reply:
Unless I miss something, here `x-pack/plugins/alerting/server/routes/bulk_enable_rules.ts` we do `body: { ...bulkEnableResults, // rest of stuff }` where `bulkEnableResults = await rulesClient.bulkEnableRules({ filter, ids });`. The response from `bulkEnableRules` is in camel case so the route returns camel case. You can see it from the tests that we were returning camel case and now we are returning snake case. Given that the API is internal I think is fine. I just wanted to be sure we are not introducing a breaking change anywhere.

Reply:
Yes thankfully we never use the return value of `bulkEnableRules`.